Umbraco-Uno/Working-with-Forms/Umbraco-Forms-widget/index.md

```diff
@@ -6,12 +6,15 @@ versionFrom: 8.0.0
 
 There are multiple ways of adding a form to your content pages in Umbraco Uno. In this guide, you will learn of two ways to add forms to a site through the use of widgets.
 
+Here is an example of what forms could look like on the front end.
+![Front end example](images/Frontend-example-forms.png)
+
 The first option is to add the widget called Umbraco Forms. In this widget, you will get the option to select which form you want to add. You can read about this widget's settings in the article about the [Umbraco Forms widget](../../Widgets/Umbraco-Form).
 
-![Regular Umbraco Forms](images/Regular-form.png)
+![Regular Umbraco Forms](images/Umbraco-forms.png)
 
 The second option is to add the form through a Grid widget. The setup here is slightly different than with the Umbraco Forms widget. However, the way you add the form is the same as the first option. Go to the section of the widget that says ***Form*** and then click "Add" to select the form you want.
 
-![Grid Umbraco Forms](images/Grid-forms.png)
+![Grid Umbraco Forms](images/Grid-form-filled.png)
 
```

Comment:
You say "in the next article", but is it going to be on the same page? If the link just before that points to the article you mentioned  I would just remove "from your form which is the next article" part